apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts

```diff
@@ -88,7 +88,10 @@ interface IPageWrapper {
     isSmall: boolean
 }
 
-export const StyledPageWrapper = styled(Layout.Content)<IPageWrapper>`
+export const StyledPageWrapper = styled(Layout.Content, {
+    // NOTE: remove `isSmall` prop on a DOM element (avoid React warning)
+    shouldForwardProp: (prop) => prop !== 'isSmall',
+})<IPageWrapper>`
```

Comment:
Looks like a hack)

Reply:
Yes, the `styled` adds attributes to the HTML element by default. And this leads to the emergence of warning